cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -16,16 +16,17 @@
 
 #pragma once
 
-#include <io/comp/gpuinflate.h>
-#include <io/statistics/column_stats.h>
-#include <io/utilities/column_buffer.hpp>
-#include <io/utilities/hostdevice_vector.hpp>
-#include "parquet_common.hpp"
+#include "io/comp/gpuinflate.h"
+#include "io/parquet/parquet_common.hpp"
+#include "io/statistics/column_stats.h"
+#include "io/utilities/column_buffer.hpp"
+#include "io/utilities/hostdevice_vector.hpp"
```

Comment:
I thought we're supposed to use <> if the header is not in the same directory.

Reply:
And I thought we're supposed to "" for headers in the src directory.

Reply:
My impression is based on https://github.com/rapidsai/cudf/blob/branch-0.20/cpp/docs/DEVELOPER_GUIDE.md#includes, maybe it's outdated.